test/utils/helperFn.ts

```diff
@@ -50,7 +50,7 @@ export function getActiveColumnHeader() {
   return `${Number(activeElement.getAttribute('aria-colindex')) - 1}`;
 }
 
-export async function sleep(duration: number) {
+export function sleep(duration: number) {
```

Comment:
async is not needed here, it returns a promise, it's enough.

Reply:
This helper function appears to never be called. Instead, in every test, where we need to wait, we create a local function. We should use `sleep` instead of duplicating code.

https://github.com/mui-org/material-ui-x/blob/6208886dbb143f7365f8387ef9d91a6261a3a9d3/test/e2e/index.test.ts#L4